silkworm/silkrpc/commands/eth_api.cpp

```diff
@@ -1245,14 +1248,14 @@ awaitable<void> EthereumRpcApi::handle_eth_create_access_list(const nlohmann::js
         reply = make_json_error(request["id"], 100, error_msg);
         co_return;
     }
-    auto call = params[0].get<Call>();
-    const auto block_number_or_hash = params[1].get<BlockNumberOrHash>();
-
-    SILK_DEBUG << "call: " << call << " block_number_or_hash: " << block_number_or_hash;
-
+    
     auto tx = co_await database_->begin();
 
     try {
+        auto call = params[0].get<Call>();
```

Comment:
`const auto bundles = params[0].get<Bundles>();` within `EthereumRpcApi::handle_eth_call_many` below could also throw now. That too would need to be put into the try catch.

Reply:
fixed